odoo/addons/base/static/tests/base_settings_tests.js

```diff
@@ -157,6 +157,44 @@ QUnit.module('base_settings_tests', {
         form.destroy();
     });
 
+    QUnit.test('hide / show setting tips properly', async function (assert) {
+        assert.expect(3);
+
+        const form = await createView({
+            View: BaseSettingsView,
+            model: 'res.config.settings',
+            data: this.data,
+            arch: `
+                <form string="Settings" class="oe_form_configuration o_base_settings">
+                    <div class="o_panel">
+                        <div class="setting_search">
+                            <input type="text" class="searchInput" placeholder="Search..." />
+                        </div>
+                    </div>
+                    <div class="o_setting_container">
+                        <div class="settings_tab" />
+                        <div class="settings">
+                            <div class="notFound o_hidden">No Record Found</div>
+                            <div class="app_settings_block" string="CRM" data-key="crm">
+                                <h2>Setting Header</h2>
+                                <h3 class="o_setting_tip">CRM settings will appear below</h3>
+                            </div>
```

Comment:
picky std: I would not use the term "CRM" in the base module, maybe just "Setting" ?

Reply:
I was trying to be consistent with the file.